src/components/table/table.tsx

```diff
@@ -189,23 +205,6 @@ class Table<IDataCoordinates = any> extends React.Component<ITableProps<IDataCoo
     this.groupsDepth = depth;
   };
 
-  private getFixedRowsIndexes = (openedTrees: ITrees, relativeIndexesMapping: IRelativeIndexesMap) => {
```

Comment:
since `getDerivedStateFromProps` is static, I had to put it out of the class (it's moved to `utils`)